config-templates/content_indices.properties

```diff
@@ -1,4 +1,4 @@
 #
 #Mon Jan 29 14:17:14 BST 2023
-latest=05bbbb74fd52cd0aea3a048a3c1e6c434ce5efaf
```

Comment:
Is this change supposed to be included?

Reply:
Yes this was necessary to pick up the new email template